src/test/java/smile/identity/core/SignatureTest.java

```diff
@@ -0,0 +1,48 @@
+package smile.identity.core;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Date;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SignatureTest {
+	
+    private String PARTNER_ID = "212";
+    private String API_KEY = "gNra9o0gy3pRFg0xc3NXZwAYtAPj1L9iOIQjX3yK0Ui2QG1lZ68VcNNfs34QW1h5Wg+PTe2UCZUjLpK5VNjwmU2I94Ihdjb9HplUODatu5QuThDmg+w7kd5PZrkXXI79xRRD6d9b7oqefA07rj3GoMFZ2YCDNN50EMPIgSdWSnw=";
```

Comment:
Partner info please remove from git history as well should not ever be part of this repo

Reply:
👍🏽